Examples/Framework/ML/src/NeuralCalibrator.cpp

```diff
@@ -48,7 +48,8 @@ std::size_t fillChargeMatrix(Array& arr, const ActsExamples::Cluster& cluster,
     // Translate each pixel
     int iMat = cell.bin[0] - offset0;
     int jMat = cell.bin[1] - offset1;
-    if (iMat >= 0 && iMat < (int)size0 && jMat >= 0 && jMat < (int)size1) {
+    if (iMat >= 0 && iMat < static_cast<int>(size0) && jMat >= 0 &&
```

Comment:
I am not completely sure, that we really need the cast here at all or if we could refactor the rest to skip a cast

Reply:
We could get rid of the cast here, but would need to do it at some place else. So let's keep it here.